src/extension/build/devtools.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
Instead of disabling this rule, perhaps we can do something like:
https://stackoverflow.com/questions/48584556/eslint-chrome-is-not-defined-no-undef

Will that work?

Reply:
should work, I guess. I'll give it a try